modules/services/getmail.nix

```diff
@@ -0,0 +1,61 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.getmail;
+
+  accounts = filter (a: a.getmail.enable)
+    (attrValues config.accounts.email.accounts);
+
+  renderConfigFilepath = a: ".getmail/getmail${a.name}";
+  quoted = concatMapStrings (x: " --rcfile \${HOME}/${x}") (map renderConfigFilepath accounts);
+  renderStartCmd = accounts: "${pkgs.getmail}/bin/getmail ${quoted}";
```

Comment:
The `accounts` argument is unnecessary. In fact, could inline this directly into the `ExecStart` line below 🙂

Reply:
Nice! Done.